src/node/NodeOptions.hs

```diff
@@ -222,6 +207,25 @@ addrNodeOption =
         metavar "HOST:PORT" <>
         help "Identifier of a node in a Kademlia network"
 
+tlsParamsOption :: Parser TlsParams
```

Comment:
I see this https://github.com/input-output-hk/cardano-sl/blob/master/lwallet/WalletOptions.hs#L97